std/c/openbsd.zig

```diff
@@ -0,0 +1,15 @@
+extern "c" fn __errno() *c_int;
```

Comment:
Just to double check- what methodology are you using to get these definitions?

Reply:
I checked in `/usr/include/errno.h` which define errno this way:

```
#ifndef errno
int *__errno(void);
#define errno (*__errno())
#endif /* errno */
```

Reply:
OK great, that's perfect. Note that you can use `zig translate-c /usr/include/errno.h` (or any .h file, or an .h file that includes a bunch of .h files) to spit out Zig declarations. It will figure out some macros too, although I'm not sure it's sophisticated enough (yet) to figure out that errno one.

The auto translated declarations often still require "cleanup" - e.g. if it's a pointer sometimes it should be changed be non-null, or single-item.

Reply:
Great ! It is capable of finding this prototype:

    pub extern fn __errno() ?[*]c_int;

Reply:
Great indeed - and that's exactly the cleanup I mentioned above - which you already did, changing `?[*]c_int` to `*c_int`. Also adding `"c"` as you did :+1: that will help at least with #1698